### DISCUSSION TITLE
NvChad and inlay hints

### DISCUSSION THREAD

**Discussion by serzhiio** [mrcjkb/rustaceanvim#432] [Q&A] [answered]
Just installed NvChad + nvim 0.10 + rust-analyzer.
With standard settings there are no inlay hints visible, is it normal?
My `init.lua` file:
```
return {
  {
    "stevearc/conform.nvim",
    config = function()
      require "configs.conform"
    end,
  },
  {
    "mrcjkb/rustaceanvim",
    version = "^4",
    lazy = false,
  },
  {
    'saecki/crates.nvim',
    ft = {"toml"},
    config = function()
      require('crates').setup{
        lsp = {
          enabled = true,
          actions = true,
          completion = true,
          hover = true,
        },
        completion = {
          crates = {
            enabled = true,
            max_results = 10,
            min_chars = 3,
          },
        }
      }
    end,
  },
}
```

**Comment by mrcjkb** [comment]
Hey :wave: 

In Neovim, inlay hints have to be enabled manually.
See [`:h lsp-inlay_hint`](https://neovim.io/doc/user/lsp.html#lsp-inlay_hint)

**Reply by serzhiio** [reply]
Thx for quick reply, i am new to neovim. 
Is there any example of config with this option turned on?

**Reply by mrcjkb** [reply, marked as answer]
I don't have it enabled by default, but I have a keymap to toggle it: https://github.com/nix-community/kickstart-nix.nvim/blob/37bc6855bde7ee36bf4c08dac0c24009f30d47c2/nvim/plugin/autocommands.lua?plain=1#L99

**Reply by 9mm** [reply]
<img width="462" alt="image" src="https://github.com/mrcjkb/rustaceanvim/assets/1386207/adf0932e-437d-4fea-9f8b-def08d26891a">
do you know what the heck these symbols mean? lol i just wanted to see return types

**Reply by mrcjkb** [reply]
That's probably caused by another plugin.
